torch/_inductor/codegen/wrapper.py

```diff
@@ -1544,75 +1622,9 @@ def define_user_defined_triton_kernel(self, kernel, configs, kwargs):
             @triton.jit
             """
         )
-        compile_wrapper.splice(kernel.src, strip=True)
-
-        # Also include any possible kernel being called indirectly
-        from triton import JITFunction  # type: ignore[name-defined, attr-defined]
-        from triton.language import constexpr  # type: ignore[name-defined]
-
-        # global constexpr vars handled above
-        symbols_included = {original_name}
-
-        def traverse(cur_kernel):
```

Comment:
any chance of doing this code move as a refactor pr stacked before this? Hard to review like this

Reply:
done